sdk/python/localcluster/cluster.py

```diff
@@ -16,7 +16,7 @@
 )
 from .node import Node
 
-GLOBAL_TIMEOUT = 90
+GLOBAL_TIMEOUT = 400
```

Comment:
On my machine, it seems 200 seconds could be sufficient. Isn't 400 too much already?

Reply:
Changed to 200 and CI passed